heimctl/retention/metrics.go

```diff
@@ -0,0 +1,15 @@
+package retention
+
+import "github.com/prometheus/client_golang/prometheus"
+
+var (
+	roomHasExpiredMsg = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Name:      "room_has_expired",
+		Subsystem: "retention",
+		Help:      "Whether a room has expired messages, labeled by room name.",
+	}, []string{"room"})
+)
```

Comment:
Could you add a gauge for the time.Now().Unix() of the most recently completed run of the scan loop?